scripts/globals/items/castellanus_cell.lua

```diff
@@ -1,4 +1,8 @@
 -----------------------------------------
+-- Castellanus Cell
+-- ID 5366
+-- Unlocks head and neck equipment
+-----------------------------------------
 require("scripts/globals/status")
```

Comment:
+require("scripts/globals/zone")

Reply:
tried it your way and got a console error, but my way works. took the example from and file in the global section that uses ID from the current zone.
![image](https://user-images.githubusercontent.com/35616771/47997486-a1a42800-e0c1-11e8-940a-875e307f85d2.png)